include/Strain2D.h

```diff
@@ -105,8 +105,8 @@ namespace ncorr {
 				double tr = d_exx + d_eyy;
 				double eigenfirst = tr / 2 + std::sqrt(std::pow(d_eyy-d_exx, 2.0) / 4 + std::pow(d_exy, 2.0));
 				double eigensecond = tr / 2 - std::sqrt(std::pow(tr, 2.0) / 4 + std::pow(d_exy, 2.0));
-				*i_e1 = std::max(eigenfirst, eigensecond);
-				*i_e2 = std::min(eigenfirst, eigensecond);
+				*i_e1 = std::abs(eigenfirst)>std::abs(eigensecond)? eigenfirst: eigensecond;

```

Comment:
Was this a bug?

Reply:
yeah, it only mattered if you did compression. Tension results were correct.

Reply:
Ok, and this was only used for tension. Makes sense